app/controllers/api/base_controller.rb

```diff
@@ -31,13 +31,13 @@ def error_response(type, error_hash)
     # Type is an arbitrary URI identifying the error type
     # https://tools.ietf.org/html/rfc7807#section-3.1 recommends using
     # human-readable documentation for this, so point to our API docs.
-    error_hash[:type] = "https://content-performance-api.publishing.service.gov.uk/errors/##{type}"
+    error_hash[:type] = "https://content-performance-api.publishing.service.gov.uk/errors.html/##{type}"
     render json: error_hash, status: :bad_request, content_type: "application/problem+json"
   end
 
   def not_found_response
     response_hash = {
-      type: "https://content-performance-api.publishing.service.gov.uk/errors/#base-path-not-found",
+      type: "https://content-performance-api.publishing.service.gov.uk/errors.html/#base-path-not-found",
```

Comment:
Would this need to be `https://content-performance-api.publishing.service.gov.uk/errors.html#base-path-not-found`?

Reply:
Good catch! Updated in c1f3100